server/app/services/applicant/question/PhoneQuestion.java

```diff
@@ -20,9 +21,8 @@
 public final class PhoneQuestion extends Question {
 
   private Optional<String> phoneNumberValue;
-  private Optional<String> countryCodeValue;
-
   private static final PhoneNumberUtil PHONE_NUMBER_UTIL = PhoneNumberUtil.getInstance();
+  private Optional<String> countryCodeValue;
```

Comment:
Question (non-blocking): Does this field need to be moved?

Reply:
no cannot be. It is the Class parameter. The PhoneUtil is needed to for the getAnswerString method

Reply:
Oh sorry, I meant: It looks like this was moved from line 23 to line 25, can it be moved back to line 23?